platform/tezos/bakingbad/api.go

```diff
@@ -0,0 +1,60 @@
+package bakingbad
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+const (
+	baseURL        = "https://api.baking-bad.org"
+	defaultTimeout = time.Second * 2
+)
+
+type (
+	API struct {
+		client *http.Client
```

Comment:
we already have a package to deal with http requests

https://github.com/trustwallet/blockatlas/blob/master/pkg/blockatlas/client.go